diff --git a/config/firewalld.service.in b/config/firewalld.service.in
index c08cb36..91d5fcb 100644
--- a/config/firewalld.service.in
+++ b/config/firewalld.service.in
@@ -7,6 +7,7 @@ Conflicts=iptables.service ip6tables.service ebtables.service
 
 [Service]
 EnvironmentFile=-/etc/sysconfig/firewalld
+EnvironmentFile=-/etc/default/firewalld
 ExecStart=@sbindir@/firewalld --nofork --nopid $FIREWALLD_ARGS
 ExecReload=/bin/kill -HUP $MAINPID
 # supress to log debug and error output also to /var/log/messages
